### DISCUSSION TITLE
Windows Default Preset, restore the all default settings

### DISCUSSION THREAD

**Discussion by jgsjnr** [farag2/Sophia-Script-for-Windows#221] [Q&A] [answered]
When you select the default preset settings in Wrapper, all modifications when you run Sophia.ps1 are restored? Like when you format, or some function to change system settings continue running or the just can reset to win10 default i need reset the pc?

**Comment by BenchTweakGaming** [comment, marked as answer]
Almost every function has a 'Windows Default'. Selecting this in the preset menu chooses the 'Windows Default' for each setting in wrapper for you to make the PS script for you to run in 'Console Output'. It may require a reboot after running the script you create.
